opendevin/server/listen.py

```diff
@@ -191,6 +193,16 @@ async def get_litellm_models():
     )
     bedrock_model_list = bedrock.list_foundation_models()
     model_list = litellm_model_list_without_bedrock + bedrock_model_list
+    if config.llm.model.startswith('ollama'):
+        # TODO: detect the run mode docker or local
+        # ollama_url = 'http://localhost:11434/api/tags'
+        ollama_url = 'http://host.docker.internal/api/tags'
+        try:
```

Comment:
```python
try:
    response = requests.get(ollama_url)
    response.raise_for_status()  # Check if the request was successful
    ollama_models_list = response.json().get('models', [])
    for model in ollama_models_list:
        model_name = model.get('name')
        if model_name:
            model_list.append(f'ollama/{model_name}')
except requests.exceptions.RequestException as e:
    logger.error(f'Error getting OLLAMA models: {e}', exc_info=True)
```

Reply:
`ollama_models_list = response.json().get('models', [])`
default value is needed?

Reply:
I think @Umpire2018 suggestion is very nice, because it's good practice to use specific exceptions, the exceptions we need for the case at hand, instead of the generic Exception as much as possible. I sometimes tend to overuse Exception too, but it's usually bad practice. 😅
We have tried to keep the codebase to this standard in the past, let's do it now too.

If you see fit to log a message, instead of silent, we could add one ourselves if the list is empty.

Reply:
I agree with the comments above, and think that this can be merged in after they are addressed.

Reply:
This comment has not yet been addressed. Please do not do an over-broad catch of the exception...